docs/001_develop/03_client-capabilities/023_custom-components/05_directives.mdx

```diff
@@ -77,15 +73,17 @@ Even though the `<button>` element is part of our `<my-button>` component’s te
 ref(propertyName);
 ```
 
-- Property Name: The name of the property that will store the DOM reference in the component. The element with `ref(propertyName)` will be assigned to this property.
+- Property Name: The name of the property that stores the DOM reference in the component. The element with `ref(propertyName)` will be assigned to this property.
 
 ### Slotted
```

Comment:
Out of scope for this PR maybe but I still think that this section being the first reference of a `slot` is wrong and we should cover that concept earlier in this custom components section